### PR TITLE
Move mozilla deprecated touch events

### PR DESCRIPTION
**Pull request by hamishwillee**
This moves the Mozilla touch events (deprecated) out of the guide and below the standard [Touch events](https://developer.mozilla.org/en-US/docs/Web/API/Touch_events). Includes redirects.

This is part of fixing #2245

Note, this does not create any explicit links to the document because the events are largely irrelevant. However they can be found by search. One alternative would have been to archive this too.

@chrisdavidmills FYI

### CODE REVIEW

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -10776,6 +10776,7 @@
 /en-US/docs/Web/Guide/DOM/Whitespace_in_the_DOM	/en-US/docs/Web/API/Document_Object_Model/Whitespace
 /en-US/docs/Web/Guide/Events/Event_dispatching_example	/en-US/docs/Web/Guide/Events/Creating_and_triggering_events
 /en-US/docs/Web/Guide/Events/Touch_events	/en-US/docs/Web/API/Touch_events
+/en-US/docs/Web/Guide/Events/Touch_events_(Mozilla_experimental)	/en-US/docs/Web/API/Touch_events/Mozilla_experimental_events
```
Comment (hamishwillee) [inline files/en-us/_redirects.txt:10779]: @chrisdavidmills This is causing an error but I can't see why: https://github.com/mdn/content/runs/2172587057?check_suite_focus=true#step:6:12

Any suggestions on how to debug?

File: files/en-us/web/guide/events/index.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
 	<li>the original 'click' event,</li>
 	<li>mouse events,</li>
 	<li><a href="/en-US/docs/Web/Guide/Events/Mouse_gesture_events">mouse gesture events</a>, and</li>
-	<li>both <a href="/en-US/docs/Web/API/Touch_events">touch events</a> and the earlier <a href="/en-US/docs/Web/Guide/Events/Touch_events_(Mozilla_experimental)">mozilla experimental touch events</a>, now deprecated.</li>
+	<li>both <a href="/en-US/docs/Web/API/Touch_events">touch events</a> and the earlier <a href="/en-US/docs/Web/Guide/Events/Touch_events_(Mozilla_experimental)">Mozilla experimental touch events</a>, now deprecated/removed.</li>
```
Comment (chrisdavidmills) [inline files/en-us/web/guide/events/index.html:33]: The link to "Mozilla experimental touch events" needs updating to point the new location/slug you've given it.

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/_redirects.txt:10779]: Have you included a space to separate the to: and from: slugs? I think it is supposed to be a tab character. Yup, pretty flaky.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Reviewed; couple of bits for you to look at. Cheers @hamishwillee !

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/_redirects.txt:10779]: Yes, it is an exact copy of the tab in the line above. It also works locally.

**Comment by hamishwillee** [comment]
@chrisdavidmills I removed the redirect to verify it is the cause of the problem. It is. However it works locally, and has a tab space, so I have no idea what could be wrong with it. We could merge without it? Otherwise we will need to escalate for debugging help

**Comment by chrisdavidmills** [comment]
@peterbe can you help us here? We've got another flaky failing redirect, and we can't work out why it is the case. As @hamishwillee asserts, it works for him locally.

**Comment by peterbe** [comment]
I think what's confusing about redirects is... 

1. you might think you're supposed to manually edit the `_redirects.txt` file when in fact you should be using the `yarn content add-redirect ...` CLI tool. 
2. I suspect that the same validation that runs in GitHub CI isn't running on your local server when you type `yarn start`. So you might be unnecessarily disappointed when CI breaks after all the hard work. 

We can solve both. For example, perhaps it's better to call it `_redirect.json` since that'd make it impossible to have to distinguish between a tab and spaces. Heck, I don't even know how to type a tab character because my IDE automatically transforms it to 4 spaces (or 2 depending on the file).

The other thing we could do to make things a whole lot easier is to do what Python poetry does. They have a lock file just like `yarn.lock` and `package-lock.json`. But they have a hash within the file itself. How the hash is computed isn't important but if the CLI is the only tool in the world that knows how to make the hash, you can easily and quickly check that only the CLI was allowed to edit the file. 

It was @fiji-flo who built most of the redirects tooling and it would be great to have some of your thoughts on this. 

PS. I love that this is being tracked in the [content-team-dev-papercuts Trello board](https://trello.com/b/3YgVTxvh/mdn-content-team-dev-papercuts) because then we can find out that multiple people are suffering from the same confusions.

**Comment by peterbe** [comment]
I took my last comment [and turned it into an issue](https://github.com/mdn/yari/issues/3342) with a [PR](https://github.com/mdn/yari/pull/3343). It's not a great solution but I really think it can help with some of the confusion.

File: files/en-us/_redirects.txt [hunk 3]
```diff
@@ -10776,6 +10775,7 @@
...
```
Comment (peterbe) [inline files/en-us/_redirects.txt:10778]: Are you sure that's a `tab` character?
Also, can you delete this line and try:
```
yarn content add-redirect /en-US/docs/Web/Guide/Events/Touch_events_(Mozilla_experimental) /en-US/docs/Web/API/Touch_events/Mozilla_experimental_events
```

**Comment by hamishwillee** [comment]
The script worked thanks @peterbe . 
The space/tab was definitely right, so I think this was something to do with their being an existing redirect from that file.

@chrisdavidmills Ready for you to merge now if you're happy.

**Review by chrisdavidmills — APPROVED** [review]
OK, let's do this.

**Pull request merged**